scripts/exregional_nonvarcldanl.sh

```diff
@@ -249,28 +245,28 @@ EOF
 #
 #-----------------------------------------------------------------------
 #
-EXEC="${EXECDIR}/fv3sar_novarcldana.exe"
+EXEC="fv3sar_novarcldana.exe"
 
-if [ -f $EXEC ]; then
+if [ -f ${EXECDIR}/$EXEC ]; then
   print_info_msg "$VERBOSE" "
-Copying the noVar Cloud Analysis executable to the run directory..."
-  cp_vrfy ${EXEC} ${workdir}
+Copying the nonVar Cloud Analysis executable to the run directory..."
+  cp_vrfy ${EXECDIR}/${EXEC} ${workdir}
 else
   print_err_msg_exit "\
 The executable specified in EXEC does not exist:
-  EXEC = \"$EXEC\"
+  EXEC = \"${EXECDIR}/$EXEC\"
 Build executable and rerun."
 fi
 #
 #
 #
 #-----------------------------------------------------------------------
 #
-# Run the novar cloud analysis application.  
+# Run the non-var cloud analysis application.  
 #
 #-----------------------------------------------------------------------
 #
-$APRUN ./fv3sar_novarcldana.exe > stdout 2>&1 || print_err_msg_exit "\
+$APRUN ./${EXEC} > stdout 2>&1 || print_err_msg_exit "\
```

Comment:
APRUN is a local variable. However it is so in all ex* scripts.

Reply:
change all EXEC to exect.

Reply:
Thanks!   You intentionally use "`$exect`" not "`$exec`" , right?

Reply:
It is $exect. This comparison does not show the latest code.